web/src/pages/_app.tsx

```diff
@@ -122,10 +124,49 @@ const App = ({ Component, pageProps }: AppProps): ReactElement => {
 
   const isSeoPage = router.pathname.includes("/starter-kits");
 
+  const heroTitle = insertIndustryContent(
+    config.starterKits.hero.title,
+    pageProps.industry?.id,
+    pageProps.industry?.name
+  );
+
+  const description = insertIndustryContent(
+    config.starterKits.seo.description,
+    pageProps.industry?.id,
+    pageProps.industry?.name
+  );
+
+  const imageAlt = insertIndustryContent(
+    config.starterKits.seo.imageAltText,
+    pageProps.industry?.id,
+    pageProps.industry?.name
+  );
+
   return (
     <>
       <Head>
         <meta name="viewport" content="width=360, initial-scale=1" />
+        {isSeoPage && (
+          <>
+            <meta property="og:title" content={heroTitle} />
+            <meta property="og:description" content={description} />
+            <meta property="og:image" content="https://navigator.business.nj.gov/img/team-success.jpg" />
```

Comment:
[pebble] Do we not want to use `NEXT_PUBLIC_WEB_BASE_URL` here?

Reply:
I think we'd want that for all of these instances, correct?